### DISCUSSION TITLE
Node hostname change

### DISCUSSION THREAD

**Discussion by gfrankliu** [k3s-io/k3s#6336] [Q&A] [answered]
I am running a single node k3s cluster and noticed the cluster has node name matches hostname.
I have a need to change hostname from time to time, but each time the k3s cluster will go to weird state. It thinks the new node (new hostname) is added and old node becomes not ready.
Is there a way to install the k3s cluster so that it will NOT depend on the node hostname? Can I use a dummy name for the single node?

**Comment by brandond** [comment, marked as answer]
https://docs.k3s.io/reference/agent-config#:~:text=%2D%2Dnode%2Dname
